### PR TITLE
Add Ashwin and Josh to the maintainers list

### PR DESCRIPTION
**Pull request by BSFishy**
### Description
Add Ashwin and Josh!!
 
### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] All tests pass
  - [x] `yarn lint`
  - [x] `yarn test-unit`
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-*   @seanneumann @AMoo-Miki @rednaksi91 @KrooshalUX @BSFishy @bandinib-amzn
+*   @seanneumann @AMoo-Miki @rednaksi91 @KrooshalUX @BSFishy @bandinib-amzn @ashwin-pc @joshuarrrr
```
Comment (bandinib-amzn) [inline .github/CODEOWNERS:1]: @BSFishy Can you please check these errors? Can we fix it?

**Review by bandinib-amzn — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (BSFishy) [reply .github/CODEOWNERS:1]: Need a repo admin to give them permissions. Will be fixed when that happens

Comment (BSFishy) [reply .github/CODEOWNERS:1]: cc @seanneumann

**Review by KrooshalUX — APPROVED** [review]

**Review by seanneumann — APPROVED** [review]
W00t!

File: .github/CODEOWNERS (hunk 1 above)
Comment (seanneumann) [reply .github/CODEOWNERS:1]: https://github.com/orgs/opensearch-project/teams/oui/members They're in.

**Review by bandinib-amzn — APPROVED** [review]

**Comment by gaiksaya** [comment]
Permissions updated:

image

**Pull request merged**